recipes/zulu-openjdk/all/conanfile.py

```diff
@@ -24,8 +24,8 @@ def _jni_folder(self):
 
     def configure(self):
         if Version(self.version) < Version("11.0.12"):
-            if self.settings.arch != "x86_64":
-                raise ConanInvalidConfiguration("Unsupported Architecture.  This package currently only supports x86_64.")
+            if self.settings.arch not in ["x86_64", "armv8"]:
+                raise ConanInvalidConfiguration("Unsupported Architecture. This package currently only supports x86_64.")
```

Comment:
I'm not sure, if the error message is the right

Reply:
Nope, I'll update. Thank for reviewing!